server/src/main/java/com/cloud/consoleproxy/ConsoleProxyManager.java

```diff
@@ -36,6 +36,9 @@ public interface ConsoleProxyManager extends Manager, ConsoleProxyService {
     String ALERT_SUBJECT = "proxy-alert";
     String CERTIFICATE_NAME = "CPVMCertificate";
 
+    ConfigKey<Boolean> ConsoleProxySslEnabled = new ConfigKey<>(ConfigKey.CATEGORY_ADVANCED, Boolean.class, "consoleproxy.sslEnabled", "false",
```

Comment:
```suggestion
    ConfigKey<Boolean> ConsoleProxySslEnabled = new ConfigKey<>(ConfigKey.CATEGORY_ADVANCED, Boolean.class, "consoleproxy.ssl.enabled", "false",
```

Reply:
> ```suggestion
>     ConfigKey<Boolean> ConsoleProxySslEnabled = new ConfigKey<>(ConfigKey.CATEGORY_ADVANCED, Boolean.class, "consoleproxy.ssl.enabled", "false",
> ```

@hsato03
There is already a global configuration named `consoleproxy.sslEnabled`

This PR does not include the renaming of the configuration